includes/class-sensei-course.php

```diff
@@ -1426,6 +1426,17 @@ public function filter_my_courses( $query ) {
 		_deprecated_function( __METHOD__, '3.0.0' );
 	}
 
+	/**
+	 * Has results links check.
+	 *
+	 * @param int $course_id Course ID.
+	 *
+	 * @return boolean
+	 */
+	private static function has_results_links( $course_id ) {
```

Comment:
This PR also changes the results links check. In the certificates, for example, the filter is registered, but in some cases nothing is added.